lib/HLSL/DxilValidation.cpp

```diff
@@ -3510,8 +3595,9 @@ static void ValidateFunction(Function &F, ValidationContext &ValCtx) {
     // Shader functions should return void.
     if (isShader && !F.getReturnType()->isVoidTy())
       ValCtx.EmitFnFormatError(&F, ValidationRule::DeclShaderReturnVoid,
-                               {F.getName()});
+                               {F.getName()});    
```

Comment:
nit: trailing whitespace.  I would have expected the clang-format checker to catch this, but for some reason that didn't appear to run against this PR.  I wonder why?